lib/generators/solidus_stripe/install/install_generator.rb

```diff
@@ -3,6 +3,10 @@ module Generators
     class InstallGenerator < Rails::Generators::Base
       class_option :auto_run_migrations, type: :boolean, default: false
 
+      def add_stylesheets
+        inject_into_file "vendor/assets/stylesheets/spree/frontend/all.css", " *= require spree/frontend/solidus_stripe\n", before: '*/', verbose: true
```

Comment:
Is there a scenario where someone uses this library and `vendor/assets/stylesheets/spree/frontend/all.css` isn't present? I believe this will throw an error. Will this work if someone tries to use this library with a custom frontend?

Reply:
@jacobherrington you're right. I tested it right now by changing the name of the file to a non-existing path, such as 

```inject_into_file "vendor/fake/frontend/fake.css", " *= require spree/frontend/solidus_stripe\n", before: '*/', verbose: true```

and Thor throws an error. 

What about adding an `if` statement that checks the existence of the file? I think this works fine: 

``` 
filename = 'vendor/assets/stylesheets/spree/frontend/all.css'
if File.file? filename
  inject_into_file filename, " *= require spree/frontend/solidus_stripe\n", before: '*/', verbose: true
end
```

I already added a commit to verify it works on Travis too.

Not sure I understand what you mean with the last question. The added styles are just cosmetics, and everything works fine also without them. Stripe.js V3 makes some assumptions regarding the HTML and the JS behavior, see their [tutorial](https://stripe.com/docs/stripe-js/elements/quickstart). As long as the custom frontend respects these assumptions then it's good to go.